README.md

```diff
@@ -179,7 +179,7 @@ This action will download an Augmented SBOM file in `$RELEASE_ASSETS/sbom.json`.
 ### Code Scanning Alerts
 
 This action will export all dismissed and open alerts to a SARIF file. By
-default, this file is named `code-scanning-alerts.json` and placed in the
+default, this defeault file is named `code-scanning-alerts.json` and placed in the
```

Comment:
It may just be me, but "By default, this default file" sounds wrong 🤔 

```suggestion
default, this file is named `code-scanning-alerts.json` and placed in the
```

---
If the additional "default" is correct, let's fix the typo:
```suggestion
default, this default file is named `code-scanning-alerts.json` and placed in the
```